pytests/iroha_cli_tests/test/assets/test_register_asset_definitions.py

```diff
@@ -105,6 +105,7 @@ def test_register_asset_with_not_existing_domain(
         iroha_cli.should(have.error(Stderr.FAILED_TO_FIND_DOMAIN.value))
 
 
+# FIXME: this should not work anymore
```

Comment:
@AlexStroke this test is broken - Iroha no longer enforces restrictions on asset names.

Reply:
@alexstroke1

Reply:
Yes, they aren't failing, while they should - an assertion that a long account cannot be registered would fail.

Reply:
make an issue and resolve discussion

Reply:
Created https://github.com/hyperledger/iroha/issues/5161